components/dashboards/thumbnail-list/component.js

```diff
@@ -1,32 +1,69 @@
 import React, { PureComponent } from 'react';
+import classnames from 'classnames';
+
 import PropTypes from 'prop-types';
-import { Link } from 'routes';
 
 // styles
 import './styles.scss';
 
 class DashboardThumbnailList extends PureComponent {
-  static propTypes = { dashboards: PropTypes.array.isRequired }
+  static propTypes = {
+    dashboards: PropTypes.array,
+    onSelect: PropTypes.func,
+    user: PropTypes.bool
+  }
 
-  render() {
-    const { dashboards } = this.props;
+  static defaultProps = {
+    onSelect: () => {},
+    dashboards: [],
+    user: false
+  }
 
+  render() {
+    const { dashboards, onSelect, user } = this.props;
+    const buttonDefaultClass = 'thumbnail-list-item';
+    const buttonClass = user ? buttonDefaultClass : `${buttonDefaultClass} no-user`;
     return (
-      <ul className="c-dashboard-thumbnail-list">
-        {dashboards.map(_dashboard => (
-          <li
-            key={_dashboard.slug}
-            style={{ backgroundImage: `url(${_dashboard.photo.original})` }}
-          >
-            <Link
-              to="dashboards_detail_custom"
-              params={{ slug: _dashboard.slug }}
-            >
-              <a><span>{_dashboard.name}</span></a>
-            </Link>
-          </li>
-        ))}
-      </ul>
+      <div className="c-dashboards-gallery-list">
+        <div className="row l-row -equal-height">
+          {
+            dashboards.map(dashboard => (
+              <div
+                key={dashboard.slug}
+                className={classnames({
+                  column: true,
+                  'small-12': true,
+                  'medium-6': true,
+                  'large-3': true
+                })}
+              >
+                <button
+                  key={dashboard.slug}
+                  tabIndex="0"
+                  className={buttonClass}
+                  style={{ backgroundImage: `url(${dashboard.photo && dashboard.photo.cover})` }}
+                  onClick={() => onSelect(dashboard)}
+                >
+                  <div className="content" htmlFor={`topic-${dashboard.slug}`}>
```

Comment:
The `htmlFor` attribute content should be updated _(we should remove the "topic" prefix and replace it)_

Reply:
Fixed here: https://github.com/resource-watch/resource-watch/pull/1185/commits/4078f99678fab223b1f93fce6c7390e3a117ae5c